xla/debug_options_flags.cc

```diff
@@ -1681,6 +1682,13 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       bool_setter_for(&DebugOptions::set_xla_gpu_use_memcpy_local_p2p),
       debug_options->xla_gpu_use_memcpy_local_p2p(),
       "Whether to use memcpy for local p2p communication."));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_offloading_memory_limit_gb",
+      int64_setter_for(&DebugOptions::set_xla_gpu_offloading_memory_limit_gb),
+      debug_options->xla_gpu_offloading_memory_limit_gb(),
+      "GPU memory usage upper limit in GB to trigger host memory offloading."
```

Comment:
The usage is strange.
Right now, there is already a memory setting for that pass via scheduler_mem_limit.
There is already some flags to adjust it.
What this new version will provide?
Why not make this a bool that default to false? This way, user don't need to find the default GPU memory threshold that is set to use it. So it  would be easier to use.

Reply:
OK, I changed it to bool. I thought this memory limit could be smaller than scheduler_mem_limit in order to kick in offloading earlier. I agree it's better to have one flag to control the memory limit.

Reply:
I have to change it back to int64_t. The activation offloading memory limit is much more smaller than scheduler_mem_limit. The former could be just few tensors for offloading candidates in a module, while the latter is the whole GPU memory used for JAX. If we use the scheduler_mem_limit for offloading, we can not enable offloading.